modules/tinymce/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The default value for `end_container_on_empty_block` option has been changed to `'blockquote'` #TINY-6559
 
 ### Fixed
+- Spaces would not be added or shown on Chrome in certain situations #TINY-8588
```

Comment:
```suggestion
- Spaces would not be added correctly on some browsers when before or after a contenteditable block element #TINY-8588
```
This would also happen on Edge and I suspect Safari, so this might be better?